Modules/IO/Meta/test/itkLargeMetaImageWriteReadTest.cxx

```diff
@@ -73,17 +71,14 @@ ActualTest(std::string filename, typename TImageType::SizeType size)
 
     std::cout << "Initializing pixel values " << std::endl;
 
-    IteratorType itr(image, region);
-    itr.GoToBegin();
-
-    pixelValue = itk::NumericTraits<PixelType>::ZeroValue();
-
+    PixelType pixelValue{};
     chronometer.Start("Initializing");
-    while (!itr.IsAtEnd())
+    for (IteratorType itr(image, region); !itr.IsAtEnd(); ++itr)
```

Comment:
Please try not to mix style changes and bug fixes in one single commit. You know how much I like to see `while` loops being replaced with `for` loops, when using an iterator. 😃  But it seems off-topic here! It isn't the thing that causes undefined behavior, right?